src/annotator/components/NotebookModal.tsx

```diff
@@ -119,13 +119,14 @@ export default function NotebookModal({
       data-testid="notebook-outer"
     >
       <div className="relative w-full h-full" data-testid="notebook-inner">
-        <div className="absolute right-0 text-xl m-3">
+        <div className="absolute right-0 m-3">
           <IconButton
-            icon={CancelIcon}
             title="Close the Notebook"
             onClick={onClose}
             variant="dark"
-          />
+          >
+            <CancelIcon className="w-4 h-4" />
```

Comment:
Previously, a named text-sizing utility class was applied to the containing element. If you give an `IconComponent` to `IconButton` (the `icon` prop), it will size it to `1em` square. So, by setting `text-xl` on the parent element, the `IconButton`'s icon would get sized to `1em` relative to the `text-xl` size, which was `16px`. OK, that works, but it's obfuscated and I'd like it to be clearer. It also couples this icon sizing with whatever the "text-xl" size is.

Our desired outcome here, less coyly, is a button with an icon that is 16x16px. We can achieve this by using standard tailwind sizing classes (a sizing "unit" is `.25rem`, or, except in atypical situations, `4px`. So `4` is our desired `16px` here). `w-[16px] h-[16px]` would work here as well, FWIW.

Because we want to control the sizing of the Icon directly here, we don't pass it as a prop but instead use it directly. This alternate usage is documented in the `frontend-shared` pattern library.

Reply:
Regarding

> So 4 is our desired 16px here). w-[16px] h-[16px] would work here as well, FWIW.

My convention is to use an existing generated utility class when available, and reach for arbitrary classes (e.g `w-[16px]` — square brackets indicate arbitrary values) when there is no available generated class.